exchanges/deribit/deribit_websocket_eps.go

```diff
@@ -689,7 +682,7 @@ func (d *Deribit) WSSubmitTransferToSubAccount(symbol string, amount float64, de
 		return nil, errInvalidAmount
 	}
 	if destinationID <= 0 {
-		return nil, errors.New("invalid destination address")
+		return nil, errInvalidDestinationID
```

Comment:
From prior [nit](https://github.com/thrasher-corp/gocryptotrader/pull/1082#discussion_r1062107321) please go across entire PR and convert things like `errors.New("unacceptable api key name")` to a global representation.